modules/oc-compliance-installing.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * security/oc_compliance_plug_in/oc-compliance-plug-in-using.adoc
+
+[id="installing-oc-compliance_{context}"]
+= Installing the oc-compliance plug-in
+
+.Procedure
+
+. Extract the `oc-compliance` image to get the `oc-compliance` binary:
++
+[source,terminal]
+----
+$ mkdir oc-compliance
+----
++
+[source,terminal]
+----
+$ oc image extract registry-proxy.engineering.redhat.com/rh-osbs/openshift-oc-compliance@sha256:07d6c4ab7388584f13e7d207349d948327de7c235173f2c5dded56d7d41d1390 --path /:oc-compliance
```

Comment:
This registry location is not publicly accessible actually, so we need to specify the published repo: `registry.redhat.io/compliance/oc-compliance-rhel8`, Also, we shouldn't use a SHA tag (or else we would need to update the docs for every release).

The extract commands can be simplified to:
`$ podman run --rm --entrypoint /bin/cat registry.redhat.io/compliance/oc-compliance-rhel8 /usr/bin/oc-compliance > ~/.local/bin/oc-compliance`